pkg/conversion/doc.go

```diff
@@ -1,21 +0,0 @@
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Package conversion only exists until heapster rebases
```

Comment:
Ugh, yes, I also assumed it would break in CI if it were a problem.

@rmmh Does the batch merger not run the right test suite to catch this? I can see why it passed on the PR but I think it should have caused the batch it was in to fail?